torchvision/ops/boxes.py

```diff
@@ -291,6 +291,25 @@ def box_area(boxes: Tensor) -> Tensor:
     return (boxes[:, 2] - boxes[:, 0]) * (boxes[:, 3] - boxes[:, 1])
 
 
+def box_area_center(boxes: Tensor) -> Tensor:
```

Comment:
Thanks a lot for the PR and for the benchmarks @alperenunlu !

I agree with @AntoineSimoulin 's suggestion in https://github.com/pytorch/vision/issues/8961#issuecomment-2824908946 to not expose the new `_center()` functions, and instead just add a new `fmt` (or `in_fmt`) parameter to the existing `box_area()` and `box_iou()` functions.

This will reduce surface area of the API, so it'll be easier to find for users, and easier for us to maintain as well!

For the rest I will let @AntoineSimoulin take a look